core/arch/interp.c

```diff
@@ -79,6 +79,11 @@
 # include "../annotations.h"
 #endif
 
+#ifdef AARCH64
+#define decode decode_with_ldstex
```

Comment:
This feels dangerous: there is a lot of code in interp.c -- how do we know it should all do this bundling?  What if some of it calls decode_sizeof() or decode_next_pc()?  Is there some less blunt way to do this?  None of the existing bundles, such as Level 0 instrs and cti-short-rewrite blocks, are handled like this.